### ISSUE TITLE
Works for about a minute, then stops syncing

### ISSUE THREAD

**Issue by borekb** [czarny/Brisync#42]
Hi, nice little utility, I like the minimalistic approach yet allowing to adjust the curve. Well done!

However, the utility works for me only for about a minute. After that, the external monitor stops syncing, for example, it gets stuck at 50% (which is the "real brightness", confirmed in monitor's OSD):

<img width="248" alt="Screen Shot 2021-06-10 at 11 58 25" src="https://user-images.githubusercontent.com/101152/121505761-65f13d80-c9e3-11eb-81a4-f10606eafc62.png">

When I go to monitor's settings, the blue bar seems to indicate that it should be 100%:

<img width="362" alt="Screen Shot 2021-06-10 at 11 57 55" src="https://user-images.githubusercontent.com/101152/121505855-799ca400-c9e3-11eb-992e-77444d734ea3.png">

And indeed, my MacBook's retina display is 100% bright, but the external monitor (Samsung U28R550) isn't.

When I restart Brightsync, it again works for a short while (about a minute or two) and then it stops.

**Comment by anthonyrisinger** [comment]
I think this might be solved by https://github.com/czarny/brisync/commit/088054cff25ce191b90db22f81d1618f96f6b2e7 -- not part of recent 1.3.1 release -- I've been running a dev build for a week or more without issue, and it, combined with a a similarly unreleased wake/sleep bugfix, solved a long-standing nuisance every time my MBP woke up.

**Comment by kirach** [comment]
> I think this might be solved by [088054c](https://github.com/czarny/brisync/commit/088054cff25ce191b90db22f81d1618f96f6b2e7) -- not part of recent 1.3.1 release -- I've been running a dev build for a week or more without issue, and it, combined with a a similarly unreleased wake/sleep bugfix, solved a long-standing nuisance every time my MBP woke up.

@anthonyrisinger how do you go about installing a dev version? I was trying to find something online but no success. Does it mean I have to manually build the app from the sources?

**Comment by anthonyrisinger** [comment]
@kirach Yeah it's been a while but it's exactly that -- though I stopped using brisync because it stopped working on the latest OSX entirely -- you'll need to have Xcode installed at the very least, and then you download (clone) the code and build it. I'm a developer by day, so I probably did some things second nature to me that I'm gonna forget, but after you `git clone`, it looks like you'll need to `git submodule update --init`.
